config/locales/models/en.yml

```diff
@@ -180,6 +180,7 @@ en:
               accepted: Sorry, you need to consent to the processing of your personal data in order to sign up.
               format: "%{message}"
             login:
+              admin_must_use_default: must use the default, please contact your chairs to use something else.
```

Comment:
Can we get rid of the comma splice here? `admin_must_use_default: must use the default. Please contact your chairs to use something else.`